kloppy/infra/serializers/event/wyscout/deserializer_v3.py

```diff
@@ -110,24 +111,74 @@ def _flip_point(point: Point) -> Point:
     return Point(x=100 - point.x, y=100 - point.y)
 
 
-class ShotZoneResults(str, Enum):
-    GOAL_BOTTOM_LEFT = "glb"
-    GOAL_BOTTOM_RIGHT = "grb"
-    GOAL_BOTTOM_CENTER = "gb"
-    GOAL_CENTER_LEFT = "gl"
-    GOAL_CENTER = "gc"
-    GOAL_CENTER_RIGHT = "gr"
-    GOAL_TOP_LEFT = "glt"
-    GOAL_TOP_RIGHT = "grt"
-    GOAL_TOP_CENTER = "gt"
-    OUT_BOTTOM_RIGHT = "obr"
-    OUT_BOTTOM_LEFT = "olb"
-    OUT_RIGHT = "or"
-    OUT_LEFT = "ol"
-    OUT_LEFT_TOP = "olt"
-    OUT_TOP = "ot"
-    OUT_RIGHT_TOP = "ort"
-    BLOCKED = "bc"
+class ShotZoneResults(Enum):
+    """
+    Wyscout does not provide end-coordinates of shots. Instead shots on goal
+    are tagged with a zone. The zones and corresponding y and z-coordinates are depicted below.
+
+                                       (z-coordinate of zone or post)
+
+        olt      | ot |      ort          3.5
+     --------------------------------
+          ||=================||           2.77
+     -------------------------------
+          || glt | gt | grt ||            2
+     --------------------------------
+      ol || gl | gc  | gr || or           1
+     --------------------------------
+      olb || glb  | gb | grb || orb       0
+
+      40     45    50    55     60    (y-coordinate of zone)
+        44.62               55.38     (y-coordinate of post)
+    """
```

Comment:
You can document the attributes here:
```suggestion
   Attributes:
          code (str): The string identifier of the shot result.
          point (Point3D): The predefined Point3D associated with the shot result.
    """
```